### ISSUE TITLE
Python 3.9 support for sql-cli

### ISSUE THREAD

**Issue by joshuali925** [opendistro-for-elasticsearch/sql#1052]
Upgraded Python to 3.9.1 and it seems to be incompatible with sql-cli. sql-cli should still work for python versions below 3.8.

this method is removed in 3.9: https://docs.python.org/3.8/library/base64.html#base64.encodestring

```
❯ odfesql
Traceback (most recent call last):
  File "/Users/<user>/venv/bin/odfesql", line 5, in <module>
    from odfe_sql_cli.main import cli
  File "/Users/<user>/venv/lib/python3.9/site-packages/odfe_sql_cli/main.py", line 21, in <module>
    from .esconnection import ESConnection
  File "/Users/<user>/venv/lib/python3.9/site-packages/odfe_sql_cli/esconnection.py", line 22, in <module>
    from elasticsearch import Elasticsearch, RequestsHttpConnection
  File "/Users/<user>/venv/lib/python3.9/site-packages/elasticsearch/__init__.py", line 24, in <module>
    from .client import Elasticsearch
  File "/Users/<user>/venv/lib/python3.9/site-packages/elasticsearch/client/__init__.py", line 5, in <module>
    from ..transport import Transport
  File "/Users/<user>/venv/lib/python3.9/site-packages/elasticsearch/transport.py", line 5, in <module>
    from .connection import Urllib3HttpConnection
  File "/Users/<user>/venv/lib/python3.9/site-packages/elasticsearch/connection/__init__.py", line 2, in <module>
    from .http_requests import RequestsHttpConnection
  File "/Users/<user>/venv/lib/python3.9/site-packages/elasticsearch/connection/http_requests.py", line 3, in <module>
    from base64 import decodestring
ImportError: cannot import name 'decodestring' from 'base64' (/usr/local/Cellar/python@3.9/3.9.1_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/base64.py)
```

**Comment by 5sdba** [comment]
+1

**Comment by joshuali925** [comment]
PR merged, python 3.9 support will be available in the next release
